plugins/inputs/system/system.go

```diff
@@ -25,10 +28,12 @@ func (_ *SystemStats) SampleConfig() string {
 	return `
   ## Uncomment to remove deprecated metrics.
   # fielddrop = ["uptime_format"]
+  ## Print out when utmp is not available for n_users
+  # warn_on_missing = true
```

Comment:
I think we don't need an option, let's just do it unconditionally but at a level that won't be too annoying.

Reply:
Removed